pkg/compiler/backend/common.go

```diff
@@ -41,3 +48,45 @@ func NewBackend(s string) compiler.CompilerBackend {
 	}
 	return compilerBackend
 }
+
+func RunCommand(cmd *exec.Cmd) (string, error) {
+	ans := ""
+	writer := NewBackendWriter(!config.LuetCfg.GetGeneral().ShowBuildOutput)
+
+	if config.LuetCfg.GetGeneral().ShowBuildOutput {
+		// We have realtime output from command. Quiet spinner.
+		SpinnerStop()
+	}
+
+	cmd.Stdout = writer
+	cmd.Stderr = writer
+
+	err := cmd.Start()
+	if err != nil {
+		return "", errors.Wrap(err, "Failed starting build")
+	}
+
+	err = cmd.Wait()
+	if err != nil {
+		return "", errors.Wrap(err, "Failed waiting for building command")
+	}
+
+	res := cmd.ProcessState.ExitCode()
+	if res != 0 {
+		errMsg := fmt.Sprintf("Failed building image (exiting with %d)", res)
+		if !config.LuetCfg.GetGeneral().ShowBuildOutput {
+			errMsg = fmt.Sprintf("Failed building image (exiting with %d): %s",
+				res, writer.GetCombinedOutput())
+		}
+
+		return "", errors.Wrap(err, errMsg)
+	}
+
+	if config.LuetCfg.GetGeneral().ShowBuildOutput {
```

Comment:
By the way, I don't see a way to supply this option by CLI, am I missing something?

Reply:
this option is under the viper config. You can do this with env: `LUET_CONFIG_GENERAL__SHOW_BUILD_OUTPUT` or through luet config (luet.yaml)
https://github.com/mudler/luet/blob/master/contrib/config/luet.yaml#L36

Reply:
Add a CLI option please, or make it enabled when ```-debug``` is supplied ( see the original issu #164 )

Reply:
and that option should be ```--live-output``` under `build`, if not tight to `--debug`. No need to repeat "build" if it's only used while building, and either "show" is not representative of what's doing (the ouput is always "showed")

Reply:
The output is not always showed if debug is not enable. The origin was to show the output also when you have the debug disable.
I will add `--live-output` on the build command then.